app/scripts/metamask-controller.js

```diff
@@ -1531,7 +1531,7 @@ export default class MetamaskController extends EventEmitter {
         },
       },
       env: {
-        isAccountSyncingEnabled: true,
+        isAccountSyncingEnabled: isManifestV3,
```

Comment:
Yep this is accepted, we will only support syncing initailly on MV3 (much nicer UX and not UI thread blocking)